lms/templates/navigation.html

```diff
@@ -97,7 +97,7 @@ <h2 class="course-header"><span class="provider">${course.display_org_with_defau
         </ul>
       </li>
     </ol>
-      % if should_display_shopping_cart_func(): # see shoppingcart.context_processor.user_has_cart_context_processor
+      % if should_display_shopping_cart_func() and not microsite.is_request_in_microsite(): # see shoppingcart.context_processor.user_has_cart_context_processor
```

Comment:
@ziafazal @mattdrayer I think to achieve what you and @griffresch want, this should just be '.... and not course'. Course should be None if navigation.html is rendered while not in a course (and should still show the shoppingcart button).